elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreAliasDefinition.java

```diff
@@ -214,16 +170,17 @@ protected void performRuntime(OperationContext context, ModelNode operation, Res
          */
         @Override
         public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
-            transformOperationAddress(operation);
-            super.execute(context, operation);
+            ModelNode transformedOperation = transformOperation(context, operation);
+            if(!CASE_SENSITIVE.resolveModelAttribute(context, context.readResourceFromRoot(context.getCurrentAddress().getParent()).getModel()).asBoolean()) {
```

Comment:
context.readResourceFromRoot(....., false) to save cloning children that aren't needed.